src/actions/make/_make.lua

```diff
@@ -178,9 +178,21 @@
 -- Format a list of values to be safely written as part of a variable assignment.
 --
 
-	function make.list(value)
+	function make.list(value, quoted)
+		quoted = false
```

Comment:
Is that the line was intentionally kept or is it a test relict ? If your intention was to suppress the new behavior, you should just have to revert make.list as it was before. No ?